config/api.yml

```diff
@@ -88,6 +88,29 @@
        :post:
        - :name: create
          :identifier: alert_status_action_new
+  :alert_definitions:
+    :description: Alert Definitions
+    :identifier: alert_definition
+    :options:
+    - :collection
+    :verbs: *gpd
```

Comment:
delete action is on a resource action, L113:
https://github.com/ManageIQ/manageiq/pull/13967/files#diff-15502b585d5033aa8043cc26237a06baR113